tests/render/test_column_section_renderer.py

```diff
@@ -85,7 +85,18 @@ class ExpectColumnAgesToBeLegalAdult(gxe.ExpectColumnValuesToBeBetween):
         min_value: int = 18
         description: str = "column values must be a legal adult age"
 
-    return ExpectColumnAgesToBeLegalAdult()
+    expectation = ExpectColumnAgesToBeLegalAdult()
+    try:
+        kwargs = expectation.configuration.kwargs
+        assert kwargs.get("column") == "ages"
+        assert kwargs.get("min_value") is None
+        assert kwargs.get("description") is None
```

Comment:
With the above logic, we should see `column` put back into `configuration.kwargs` but the other two overriden params left alone